core/src/components/tab-bar/tab-bar.ios.vars.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../themes/ionic.globals.md";
+
+// iOS Tab Bar
+// --------------------------------------------------
+
+/// @prop - Alpha of translucent tab bar background color
+$tab-bar-ios-translucent-background-color-alpha:    .8 !default;
+
+/// @prop - Translucent tab bar background color
+$tab-bar-ios-translucent-background-color:          rgba($background-color-rgb, $tab-bar-ios-translucent-background-color-alpha) !default;
+
+/// @prop - Filter of the translucent tab bar background color
+$tab-bar-ios-translucent-filter:                    saturate(180%) blur(20px) !default;
```

Comment:
Was this an intentional change?

```
saturate(210%) blur(20px);
```

Reply:
Yeah -- we're using `180%` in other places, so I changed it to be consistent

Reply:
I think `210%` was set by ben, may want to check with him prior to this